drivers/memc/memc_stm32.c

```diff
@@ -59,5 +59,7 @@ static const struct memc_stm32_config config = {
 	.pinctrl_len = ARRAY_SIZE(pinctrl),
 };
 
+#if DT_NODE_HAS_STATUS(DT_INST(0, st_stm32_fmc), okay)
```

Comment:
This is not required

Reply:
What tests should I run on the board and how do I run all of them?

Reply:
As of today this check is usually not done. Only multi-instance drivers take into account "okay" devices to initialize the enabled ones.

See for example https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/counter/counter_ll_stm32_rtc.c (single instance).

I'm not sure if there is a defined policy @erwango ?